### DISCUSSION TITLE
Email Address Separator - Comma or Semicolon or Both

### DISCUSSION THREAD

**Discussion by rohitthakral** [postalserver/postal#2907] [Help with using Postal]
Hi Guys, 

We have an application sending emails to Postal with comma separation while Postal is only accepting semi colon. Can postal be configured to accept both commas and semicolons for email address separation? 

Thanks
Rohit

**Comment by willpower232** [comment]
are you using SMTP or API to send the messages?

**Reply by rohitthakral** [reply]
SMTP

**Reply by willpower232** [reply]
is your SMTP library trying to put multiple addresses in the `RCPT TO` command? I'm not sure that should work

**Reply by rohitthakral** [reply]
We are using Redmine https://www.redmine.org/ to send emails. The emails are working fine with semicolon separated values from Redmine. These are entered manually.

The problem happens when the emails are read from office 365 they come in as comma-separated values instead of semicolon separated value and when redmine tries to send emails to those using postal, they all fail as the commas are read as part of one email address. 

I am not able to test using telnet yet because of this issue. https://github.com/postalserver/postal/issues/1060 (a different story)

**Reply by willpower232** [reply]
what error are you actually seeing? Can you send a screenshot? I'm not sure how Postal could be responsible for this problem

**Reply by rohitthakral** [reply]
Hi Will, 

Thank you for all your support and advice. 

Seems like I was chasing a ghost. The Comma separated list of emails from the application are working as good as the semicolon separated list and it must have been some other error in one of the previous emails which made me suspicious about the problems with comma and not semicolons. 

But to reconfirm, the emails are working fine and most probably Redmine is sending emails to postal as one individual email as it should.

Have a good rest of the day and thanks again!

Regards
Rohit
